brave/ui/app/pages/connect-accounts/connect-accounts.component.js

```diff
@@ -0,0 +1,175 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import BitGoLogoIcon from '../../components/app/dropdowns/assets/bitgo-logo'
+import ConnectWallet from '../../components/app/connect-wallet'
+import CloseIcon from '../first-time-flow/welcome-modal/assets/close-icon'
+import CloseIconDark from '../first-time-flow/welcome-modal/assets/close-icon-dark'
+import {
+  BRAVE_CONNECT_WALLETS_ROUTE,
+  INITIALIZE_CREATE_PASSWORD_ROUTE,
+  BRAVE_BITGO_WALLET_INDEX
+} from '../../helpers/constants/routes'
+const { supportedCoins } = require('../../../../app/scripts/controllers/bitgo')
+
+module.exports = class BraveConnectAccounts extends PureComponent {
+  static contextTypes = {
+    t: PropTypes.func,
+  }
+
+  static propTypes = {
+    history: PropTypes.object,
+    createBitGoWallet: PropTypes.func,
+    bitGoCreatedWallets: PropTypes.array,
+  }
+
+  constructor (props) {
+    super(props)
+
+    const checkedAssets = {}
+    const { bitGoCreatedWallets } = this.props
+    this.bitGoCreateableAssets = {}
+
+    for (let coin in supportedCoins) {
+      if (!bitGoCreatedWallets.includes(coin)) {
+        this.bitGoCreateableAssets[coin] = supportedCoins[coin]
```

Comment:
if i understand correctly, this is only showing assets that aren't already created as bitgo wallets as createable? FWIW bitgo supports having multiple wallets for the same asset for the same user, but i'm not sure if that's something we want to support in the UI.

Reply:
That's correct yes, as far as I saw we didn't support naming specific wallets with our proxy. I think this is fine for now, maybe in the future we can have ux around managing multiple wallets of one type cc: @jamesmudgett

Reply:
right, there is a wallet label but this is a random phrase generated by the proxy server. there's no way currently for users to change the label.